README.md

```diff
@@ -1 +1,86 @@
-# cap-tech-page
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Cap stone project - 
+
+> This is my first capstone project created with html, css, javascript and photoshop
+
+## Recent Fixes
+
+- 
+
+## Built With
+
+- HTML
+  - Use semantic HTML tags.
+  - Apply best practices in HTML code.
+
+- CSS
+  - Use CSS selectors correctly.
+  - Use CSS box model.
+  - Use Flexbox to place elements in the page.
+  - Use grid to place elements in the page.
+  - Media queries use to create UIs adaptable to different screen sizes.
+    - Mobile Max-width: 768px
+    - Desktop Min-width: 768px
+
+- JavaScript
+  - Apply JavaScript best practices and language style guides in code.
+  - Use JavaScript to manipulate DOM elements.
+  - Use JavaScript events.
+  - Use objects to store and access data.
+
+- GitHub
+  - Use GitHub Pages to deploy web pages.
+
+## Live Demo
+
+- [Live link](https://https://187jjay187.github.io/cap-tech-page/)
```

Comment:
- [ ]  as mentioned by previous reviewer, your live link is still breaking.
- [ ] Also you should add a loom video contains a clear and objective description about the project. Remember to highlight a piece of code you find interesting or you are proud of making.

Reply:
live demo link has been added to the README file: https://187jjay187.github.io/cap-tech-page/
Loom video link has been added to the README file: https://www.loom.com/share/3a9a3e10d10d429b89f2db89be4cb199